.steering/github-copilot/Agent MCP.md

```
@@ -0,0 +1,123 @@
+      "url": "https://your-notion-mcp-server.com/sse",
+      "headers": {
+        "Authorization": "Bearer $NOTION_TOKEN"
+      },
+      "tools": ["list_databases", "read_page"]
+    }
+  }
+}
+```
+
+### Server Types
+| Type | Config Keys | Example |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)